### DISCUSSION TITLE
Add riscv64 build in release

### DISCUSSION THREAD

**Discussion by jaxvanyang** [fastfetch-cli/fastfetch#996] [General]
Is there any plan for this?

**Comment by CarterLi** [comment]
What glibc version do you use?

**Reply by jaxvanyang** [reply]
2.35 for Ubuntu 22.04 would be fine.

**Comment by CarterLi** [comment]
https://github.com/fastfetch-cli/fastfetch/actions/runs/9413631875

**Reply by jaxvanyang** [reply]
Much thanks!
